Makefile-test.mk

```diff
@@ -52,14 +52,6 @@ KIND_CLUSTER_NAME ?= kind
 
 CYPRESS_IMAGE_NAME ?= cypress/base:22.14.0
```

Comment:
Created cherry pick for this

https://github.com/kubernetes-sigs/kueue/pull/5581